packages/edit-post/src/hooks/commands/use-common-commands.js

```diff
@@ -139,12 +189,32 @@ export default function useCommonCommands() {
 				toggleDistractionFree();
 			}
 			close();
+			createInfoNotice(
+				isTopToolbar
+					? __( 'Top toolbar off.' )
+					: __( 'Top toolbar on.' ),
+				{
+					id: 'core/edit-post/toggle-top-toolbar/notice',
+					type: 'snackbar',
+					actions: [
+						{
+							label: __( 'Undo' ),
+							onClick: () => {
+								toggle( 'core', 'fixedToolbar' );
+							},
+						},
+					],
+				}
+			);
 		},
 	} );
 
 	useCommand( {
 		name: 'core/toggle-code-editor',
-		label: __( 'Toggle code editor' ),
+		label:
+			editorMode === 'visual'
+				? __( 'Open code editor' )
+				: __( 'Exit code editor' ),
```

Comment:
We use "close" in other commands, but the existing control for the code editor is "Exit" — so we're using that here to maintain that consistency.